### PR TITLE
Refactored FacadeResponse

### PR DESCRIPTION
**Pull request by VenuAjitesh**
Implemented
1) Gave requestId as a response to the facade.
2) Removed consent RequestId in consent-status respose.
3) Remove health-information dateRange from health-information status.

### CODE REVIEW

File: src/main/java/com/nha/abdm/wrapper/hiu/hrp/consent/requests/callback/ConsentStatus.java [hunk 1]
```diff
@@ -12,7 +13,13 @@
...
 @AllArgsConstructor
 public class ConsentStatus {
-  private String id;
+  /**
+   * Added JsonIgnore because while fetching the consentStatus there are two id
+   * 1) consentRequestId
+   * 2) consentId in consentArtifacts both are represented as ID
+   * so the consentRequestId is not required for user, so ignoring the consentRequestId
+   */
+  @JsonIgnore private String id;
```
Comment (atulai-sg) [inline src/main/java/com/nha/abdm/wrapper/hiu/hrp/consent/requests/callback/ConsentStatus.java:22]: nit: lets have private String id in a separate line

File: src/main/java/com/nha/abdm/wrapper/hiu/hrp/dataTransfer/HIUFacadeHealthInformationService.java [hunk 2]
```diff
@@ -79,10 +89,28 @@ public FacadeResponse healthInformation(
...
+    Consent consentDetails =
+        patientService.getConsentDetails(
+            consentPatientService.findMappingByConsentId(consentId).getAbhaAddress(), consentId);
+    if (Objects.isNull(consentDetails)) {
+      return FacadeResponse.builder()
+          .clientRequestId(hiuClientHealthInformationRequest.getRequestId())
+          .error(
+              ErrorResponse.builder()
+                  .code(400)
+                  .message("ConsentId not found in consentPatient")
```
Comment (atulai-sg) [inline src/main/java/com/nha/abdm/wrapper/hiu/hrp/dataTransfer/HIUFacadeHealthInformationService.java:108]: lets give the message as "Consent not found in database."

Comment (VenuAjitesh) [reply src/main/java/com/nha/abdm/wrapper/hiu/hrp/dataTransfer/HIUFacadeHealthInformationService.java:108]: Modified to database.

File: src/main/java/com/nha/abdm/wrapper/common/responses/FacadeResponse.java [hunk 3]
```diff
@@ -10,6 +10,7 @@
 @Builder
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class FacadeResponse {
+  private String clientRequestId;
```
[the remark this answers is not in the document]
Comment (VenuAjitesh) [reply src/main/java/com/nha/abdm/wrapper/common/responses/FacadeResponse.java:13]: Modified them in both hip-facade.yaml and hiu-face.yaml.

File: src/main/java/com/nha/abdm/wrapper/hiu/hrp/consent/requests/callback/ConsentStatus.java (hunk 1 above)
Comment (VenuAjitesh) [reply src/main/java/com/nha/abdm/wrapper/hiu/hrp/consent/requests/callback/ConsentStatus.java:22]: The spotless modifies the spring annotations into a single line but modified the changes.

File: src/main/java/com/nha/abdm/wrapper/common/responses/FacadeResponse.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (atulai-sg) [reply src/main/java/com/nha/abdm/wrapper/common/responses/FacadeResponse.java:13]: umm.. I dont see any new commit with these changes

File: src/main/java/com/nha/abdm/wrapper/hiu/hrp/dataTransfer/requests/HIUClientHealthInformationRequest.java [hunk 4]
```diff
@@ -13,7 +13,4 @@
 public class HIUClientHealthInformationRequest {
   private String requestId;
   private String consentId;
-  private String fromDate;
```
Comment (atulai-sg) [inline src/main/java/com/nha/abdm/wrapper/hiu/hrp/dataTransfer/requests/HIUClientHealthInformationRequest.java:16]: lets make these changes in `hiu-facade.yaml` and `HIUController.java` as well

File: sample-hip/specs/hip-facade.yaml [hunk 5]
```diff
@@ -186,6 +186,8 @@ components:
     FacadeResponse:
       type: object
       properties:
+        clientRequestId:
+          type: String
```
Comment (atulai-sg) [inline sample-hip/specs/hip-facade.yaml:190]: if we are making any change in spec, then we should generate corresponding code and raise pr

Comment (VenuAjitesh) [reply sample-hip/specs/hip-facade.yaml]: Generated sample files

File: src/main/java/com/nha/abdm/wrapper/hiu/hrp/dataTransfer/requests/HIUClientHealthInformationRequest.java (hunk 4 above)
Comment (VenuAjitesh) [reply src/main/java/com/nha/abdm/wrapper/hiu/hrp/dataTransfer/requests/HIUClientHealthInformationRequest.java:16]: Modified the changes.

**Review by atulai-sg — APPROVED** [review]

**Pull request merged**